Code/Editor/NewLevelDialog.cpp

```diff
@@ -105,10 +105,74 @@ CNewLevelDialog::CNewLevelDialog(QWidget* pParent /*=nullptr*/)
     ReloadLevelFolder();
 }
 
+
+
 CNewLevelDialog::~CNewLevelDialog()
 {
 }
 
+void CNewLevelDialog::InitTemplateListWidget() const
+{
+    ui->listTemplates->clear();
+
+    const QIcon defaultIcon(":/NewLevel/res/Prefab_80.svg");
+    const QSize iconSize(128, 128);
+    const QString fileFilter = {"*.prefab"};
+    QStringList templatePaths =
+    {
+        {QString(AZ::Utils::GetEnginePath().c_str()) + QDir::separator() + "Assets/Editor/Prefabs"}
+    };
+    QStringList allTemplateFiles;
+
+    // Try to read a custom template path from the settings registry.
+    if(const AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry)
```

Comment:
nitpick: There should be space between the if and the opening parenthesis.
```suggestion
    if (const AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry)
```